spec/features/admin/gdpr/requests_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'GDPR requests', type: :feature, js: true do
+  include Warden::Test::Helpers
+
+  let(:admin_user) { create(:admin_user) }
+
+  before { login_as admin_user }
+
+  context 'when there are not requests' do
+    before { visit spree.admin_gdpr_requests_path }
+
+    it 'shows empty message' do
+      expect(page).to have_selector('.no-objects-found')
+    end
+  end
+
+  context 'when there are requests' do
+    context 'when there are less than 25 requests' do
```

Comment:
You should use _fewer_ instead than _less_ here. _Fewer_ is for countable nouns, while _less_ is for uncountable ones.